src/centralserver/admin-ui/vue.config.js

```diff
@@ -34,4 +34,13 @@ module.exports = {
       },
     },
   },
+  chainWebpack: config => {
+    config
+      .plugin('fork-ts-checker')
+      .tap(args => {
+        //args[0].memoryLimit = 4096;
+        //args[0].workers = 2;
+        return args
```

Comment:
Is this some hint for future use cases or should this '.tap - ...' statement actually be removed for it's just commented out code?